### ISSUE TITLE
(Solved) Black screen when connecting, Wrapper fully supported ver. 10.0.18362.267

### ISSUE THREAD

**Issue by booker27** [stascorp/rdpwrap#875]
Windows updated to version 1903 yesterday, so I used the autoupdater from #860 and the Wrapper got fully supported again, however when I use RDPCheck it just gives me a black screen and it shuts down after a while.

![Sem título2](https://user-images.githubusercontent.com/54714203/64027362-e8fb1980-cb16-11e9-880f-82e4ca012c1e.png)

So I tested it in a laptop we use regularly and it connects just fine, even shows up in the task manager, but on the laptop it still just a black screen, though there it only shuts down when I disconnect the user.

![Sem título](https://user-images.githubusercontent.com/54714203/64027354-e698bf80-cb16-11e9-9bbb-3c92a703bb43.png)

I tried reinstalling it from scratch and still no dice.

**Comment by s0nic9** [comment]
it's windows new feature problem not rdpwrap issue https://github.com/stascorp/rdpwrap/issues/858#issuecomment-523192537

**Comment by booker27** [comment]
Thanks! @s0nic9 it's working now, didn't even need to reboot.

Funny thing is I checked issue #858 but since I wasn't reconnecting I dind't check your solution, but now that I'm thinking about it, I logged in locally into every user in the morning so they wouldn't have to go through lenghty that update introduction Windows does, and then disconnected, so F me for being nice I guess. Thanks a lot for the quick response.
